base/shell/explorer/syspager.cpp

```diff
@@ -181,6 +181,7 @@ class CNotifyToolbar :
 
 class CHardErrorThread
 {
+    DWORD m_ThreadId;
     LONG m_bThreadRunning;
```

Comment:
Technically you can use BOOL, and since BOOL has the very same size as LONG on Windows on any x86 or x64, then the Interlocked functions should be fine too.

Reply:
This does not solve a problem.

Reply:
That's exactly Mark's point I think. Your suggestion seems to be a solution to something... but the problem you're trying to solve with it is not obvious.

Reply:
Huh ^^ No problem being solved actually! It was just about using BOOL type for a boolean actually!